css-scss/mainPage.css

```diff
@@ -0,0 +1,291 @@
+.topContainer {
+  background: url(caffee2.jpg) no-repeat center fixed;
+  height: 550px;
+  background-size: cover;
+  width: 100%;
+  z-index: 1;
+  position: relative;
+  text-align: center;
+}
```

Comment:
you should leave an empty row after these blocks. You have on some of them, so now it's not consistent :D